cookbooks/continental_extension/doc/continental_extension.md

```diff
@@ -29,11 +29,10 @@ The 2D Cartesian model spans 200 ($x$) by 100 ($y$) km and has a finite
 element grid with 1.25 and 2.5 km grid spacing, respectively, above and below
 50 km depth. This variation in grid spacing is achieved with a single initial
 adaptive refinement step using the minimum refinement function strategy.
-Unlike the crustal deformation cookbook (see
-Section&nbsp;{ref}`sec:cookbooks-crustal-deformation`, the mesh is not
-refined with time.
+Unlike the crustal deformation cookbook (see {ref}`sec:cookbooks-crustal-deformation`),
```

Comment:
{ref}`sec:cookbooks:crustal-deformation` is how that section header is currently formatted, might have been recently changed